docs/basics/RootSaga.md

```diff
@@ -0,0 +1,84 @@
+# Setting up your root Saga
+
+In the beginner tutorial it is shown that your root saga will look something like this
+
+```javascript
+export default function* rootSaga() {
+  yield all([
+    helloSaga(),
+    watchIncrementAsync()
+  ])
+}
+```
+
+This is really just one of a few ways to implement your root. Here, the `all` effect is used with an array of your sagas in a non-blocking fashion. Other root implementations may help you better handle errors and more complex data flow.
+
+Contributor @slorber mentioned in issue #760 several other common root implementations. To start, there are some other implementations you may see that will behave similarly to the  tutorial root saga behavior:
+
+```javascript
+export default function* root() {
+  yield [
+    fork(saga1),
+    fork(saga2),
+    fork(saga3)
+  ]
+}
+```
+or 
+
+```javascript
+export default function* root() {
+  yield fork(saga1)
+  yield fork(saga2)
+  yield fork(saga3)
+}
+```
+
+While making use of `fork` rather than `all`, the behavior in your app is the same: all of your sub-sagas are started. The difference between these examples and the original implementation is in execution under the hood. `fork` is non-blocking and so allows the `rootSaga` in these two cases to finish while the child sagas are kept running and blocked by their internal effects.
```

Comment:
I would probably explain that `all([fork, fork, fork])` gives back a single effect (`all`) to be handled to the saga runtime and `fork fork fork` gives a single fork effect 3 times. Ofc ultimately it doesn't matter and both would execute those effects in the same order etc.